ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/DataHubPlugin.groovy

```diff
@@ -135,6 +139,18 @@ class DataHubPlugin implements Plugin<Project> {
 
         dataHub = DataHub.create(hubConfig)
         project.extensions.add("dataHub", dataHub)
+
+        // DHF never needs the default REST server provided by ml-gradle
+        appConfig.setNoRestServer(true)
+
+        // DHF defaults to a single config dir of the hub config dir (hub-internal-config)
+        // Add the user config dir as well
+        appConfig.getConfigDirs().add(new ConfigDir(hubConfig.getUserConfigDir().toFile()))
+
+        println "Will look for resource configuration files in the following directories:"
+        for (ConfigDir configDir : appConfig.getConfigDirs()) {
+            println "Configuration directory: " + configDir.getBaseDir()
+        }
```

Comment:
Currently, there are separate staging and final schemas and triggers dbs. The expected behavior currently is that schemas under "hub-internal-config/databases/schemas" should be deployed to the data-hub-staging-SCHEMAS and the ones under "ml-config/databases/schemas" to data-hub-final-SCHEMAS. Similar behavior is expected for triggers as well. Will that hold up by adding "ml-config" dir to the configdir list

Reply:
I'll test it locally, I don't think it should matter because I doubt that the code for loading schemas cares about mlConfigPaths (as schemas are not resources).

Reply:
@srinathgit I'm working off a branch off develop off 4.0.3 - I added files to hub-internal-config/databases/schemas and ml-config/databases/schemas, and they were ignored. Only hub-internal-config/schemas was scanned and loaded from. 

I then tried using 4.0.3 and got the same behavior. So it doesn't look like those directories are supported. I also don't see any code in DHF that would provide this behavior. I only see this in HubConfigImpl:

```
ConfigDir configDir = new ConfigDir(getUserConfigDir().toFile());
config.setConfigDir(configDir);
config.setSchemasPath(getUserSchemasDir().toString());
```

Reply:
Sorry, I got the paths wrong. It is "hub-internal-config/schemas"  for loading to data-hub-staging-SCHEMAS  and "ml-schemas" for loading to data-hub-final-SCHEMAS